scripts/config.pl

```diff
@@ -166,14 +191,22 @@
 my @config_lines = <$config_read>;
 close $config_read;
 
-my ($exclude_re, $no_exclude_re);
+# Add required baremetal symbols to the list that is included.
+if ( $action eq "baremetal" ) {
+    @non_excluded = ( @non_excluded, @non_excluded_baremetal );
+}
+
+my ($exclude_re, $no_exclude_re, $exclude_baremetal_re);
```

Comment:
There shouldn't be a separate variable `$exclude_baremetal_re`. Below, `$exclude_re` should be whatever we're excluding. However, given https://github.com/ARMmbed/mbedtls/pull/1230/files#r158299362 I'm classifying this as an acceptable maintainability issue for the time being, to be revised as part of a more general cleanup of this script.

Reply:
It is easier to do full -> bare metal compared to evaluate ```$exclude_re``` as a final exclusion regex.